permissions/src/test/java/org/odk/collect/permissions/PermissionsProviderTest.kt

```diff
@@ -269,22 +272,22 @@ class PermissionsProviderTest {
     }
 
     @Test
-    fun `isReadPhoneStatePermissionGranted() when read phone state permission is granted returns true`() {
-        whenever(permissionsChecker.isPermissionGranted(Manifest.permission.READ_PHONE_STATE)).thenReturn(true)
+    fun `isReadPhoneStatePermissionGranted() when read phone numbers permission is granted returns true`() {
+        whenever(permissionsChecker.isPermissionGranted(Manifest.permission.READ_PHONE_NUMBERS)).thenReturn(true)
 
         assertThat(permissionsProvider.isReadPhoneStatePermissionGranted, `is`(true))
     }
 
     @Test
-    fun `isReadPhoneStatePermissionGranted() when read phone state permission is not granted returns false`() {
-        whenever(permissionsChecker.isPermissionGranted(Manifest.permission.READ_PHONE_STATE)).thenReturn(false)
+    fun `isReadPhoneStatePermissionGranted() when read phone numbers permission is not granted returns false`() {
+        whenever(permissionsChecker.isPermissionGranted(Manifest.permission.READ_PHONE_NUMBERS)).thenReturn(false)
 
         assertThat(permissionsProvider.isReadPhoneStatePermissionGranted, `is`(false))
     }
 
     @Test
-    fun `requestReadPhoneStatePermission() when read phone state permission is granted calls PermissionListener#granted`() {
-        permissionsApi.setGrantedPermission(Manifest.permission.READ_PHONE_STATE)
+    fun `requestReadPhoneStatePermission() when read phone numbers permission is granted calls PermissionListener#granted`() {
```

Comment:
You can add a test for the lower API levels using `@Config(sdk = x)` here. Probably also best to add `@Config` to these tests so that it's explicit which API level is being used.

Reply:
I remove the commit with the fix for https://github.com/getodk/collect/issues/4838 and it was a part of that commit.